torch/_export/converter.py

```diff
@@ -186,6 +277,9 @@ def convert_graph_inputs(self):
                         persistent=True,
                     )
                 )
+                fx_node = get_node_for_param_and_buffer(
+                    self.fx_graph, normalized_name, self.is_top_level_graph
+                )
```

Comment:
This doesn't correctly preserve the state dict key names. For example, in the first test case with module `M`, if you `print(M().state_dict().keys())` it will return `['linear.weight', 'linear.bias']`. However, now if you `print(ep.state_dict.keys())`, it will return `['linear_weight', 'linear_bias']`.